src/idlcxx/src/streamers.c

```diff
@@ -169,27 +178,45 @@ unroll_sequence(const idl_pstate_t* pstate,
     return IDL_RETCODE_NO_MEMORY;
 
   idl_retcode_t ret = IDL_RETCODE_OK;
+  char* new_read_accessor = NULL;
+  if (maximum)
+  {
```

Comment:
Use style consistent with the rest of the code in the generator please.

Reply:
will do